src/Traits/FindMethodTrait.php

```diff
@@ -56,6 +56,35 @@ protected function _findRecord($id, Subject $subject)
     }
 
     /**
+     * Find all records
+     *
+     * @param \Crud\Event\Subject $subject Event subject
+     * @return \Cake\ORM\Entity
+     */
+    protected function _findAll(Subject $subject)
+    {
+        $repository = $this->_table();
+
+        $query = $repository->find($this->findMethod());
```

Comment:
You we have an event triggered after this? So we can modify the query before it is executed.